### ISSUE TITLE
A11y: Additional issues from VSP review

### ISSUE THREAD

**Issue by batemapf** [department-of-veterans-affairs/va.gov-team#9608]
> The only feedback I'd offer besides VSA's comments would be to test on an Android if possible. I know the team is already looking for one, so that's 💯
> 
> Ideally the text above each Yes | No input group is being read after the user selects one, and the browser scrolls to the next decision group. I have a thought that this would come in handy for a lot of users who are balancing multiple items (mental and 
> 

From: https://github.com/department-of-veterans-affairs/va.gov-team/issues/9497#issuecomment-634305259

**Comment by batemapf** [comment]
@stephenBarrs going to check with a non-sighted user that _might_ be an Android user about experience.

**Comment by stephenBarrs** [comment]
> @stephenBarrs going to check with a non-sighted user that _might_ be an Android user about experience.

The non-sighted user is not using an Android device.  We have asked the 508 group to provide feedback on Android testing and expect results this week.

**Comment by stephenBarrs** [comment]
Received the following information from Jose Beltran in 508 group regarding Android testing findings:

> I went through the covid19 screen tool using an android Motorola phone with talk back enabled. I identified two issues that stood out to me.

> 2.4.3. Focus Order- After a user answers "Yes" or "No" the next question appears. A user swipes right and focus goes to the previous "No" button
rather than staying on the displayed question.
What I would expect: After a user answers the first question and swipes right for TB to read out the next question.

> 1.3.1 Info and Relationships- Bulleted list should be announced in the following question - In the past 3 days, have you had any of these symptoms? The bulleted list was not announced.

**Comment by batemapf** [comment]
closed by #9967 and #10998
